### PR TITLE
fix instructions for contributing to lib.d.ts

### PR DESCRIPTION
**Pull request by sparecycles**
@mhegazy

### CODE REVIEW

**Comment by sparecycles** [comment]
Tried running ```jake lib``` in the #5461 commit, didn't work then either :wink:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The library sources are in: [src/lib](https://github.com/Microsoft/TypeScript/tr
 
 To build the library files, run
 ```Shell
-jake lib
+jake LKG
```
Comment (mhegazy) [inline CONTRIBUTING.md]: no, it is lib that builds the library files.

`jake lib` -> `src\lib\*.d.ts` combine them into `built\local\lib.*.d.ts`
`jake LKG` -> `jake local` with `debug=false`, copy `built\local\*` to `lib\*` in preparation of publishing an npm

so files in `\lib\` should not be touched by hand, only through `jake LKG`

**Comment by mhegazy** [comment]
> Tried running jake lib in the #5461 commit, didn't work then either :wink:

how is that? what did you see? any errors?

**Comment by sparecycles** [comment]
You don't even need ```jake lib``` to update the built/local data, just running jake after changing ```src/lib``` files updates that fine (as I expect running jake with no arguments to do).

But, when I run ```jake lib``` I don't think it's out of scope to assume ```lib``` would be updated.

In the instructions: "To build the library files" obviously means the files in ```lib/``` (except it doesn't).  The README in ```lib/``` says these files are not meant to be updated by hand, and changes should be made in ```src/``` but nowhere is it documented that the command is ```jake LKG```.

Can you not see how this is confusing?

**Comment by mhegazy** [comment]
i see.  well in my defense, `\lib` was called `\bin` and that was less confusing.. 

Any how, so we can make it `jake local` or just 'jake`, how about that?

**Comment by sparecycles** [comment]
(hmm... musing) perhaps ```/bootstrap``` ( or ```/bs``` ? ) would be a better name?

**Comment by mhegazy** [comment]
looks good. thanks!

**Pull request merged**